docs/faq.rst

```diff
@@ -119,8 +119,11 @@ adjust the level or algorithm).
 levels. Which of them is the best for you pretty much depends on your
 use case, your data, your hardware -- so you need to do an informed
 decision about whether you want to use compression, which algorithm
-and which level you want to use. This is why compression defaults to
-none.
+and which level you want to use.
+
+The default is LZ4, which is fast enough to not use significant CPU time
```

Comment:
as our option value is "lz4", maybe rather write it like that. (same below at other places)